ivy/functional/frontends/tensorflow/nn.py

```diff
@@ -123,3 +123,8 @@ def batch_normalization(x, mean, variance, offset, scale, variance_epsilon, name
     return x * ivy.astype(inv, x.dtype, copy=False) + ivy.astype(
         offset - mean * inv if offset is not None else -mean * inv, x.dtype
     )
+
+
+def compute_average_loss(per_example_loss, sample_weight=None, global_batch_size=None):
+    return ivy.compute_average_loss(per_example_loss, sample_weight=sample_weight, global_batch_size=GLOBAL_BATCH_SIZE)
```

Comment:
I don't think there's a `compute_average_loss` function in Ivy's functional API, so either you'll have to implement the `tensorflow` `compute_average_loss` frontend as a composition of existing Ivy functions or extend Ivy's functional API by adding `ivy.compute_average_loss` in a separate PR referring to the [docs](https://lets-unify.ai/ivy/contributing/4_open_tasks.html#ivy-api-extensions)